markdown/clientaccess/kerberos.html.md.erb

```diff
@@ -697,3 +753,172 @@ Follow these steps to install and configure a Kerberos KDC server on a Red Hat E
+
+8. Add a Service Principal Name (SPN) for the account just created:
+
+   ``` shell
+   PS C:\Users\Administrator> setspn -A postgres/hdp3.example.com gpadmin
+   ```
+   
+9. Generate a keytab file by using the `ktpass` command. This command takes the form:
+
+   ``` shell
+   ktpass -princ postgres/<FQDN of HDB master> -pass <password> -mapuser <Managed Service Account name> -crypto ALL -ptype KRB5_NT_PRINCIPAL -out <keytab filename> -kvno 0 
+   ```
+   For example, to create `hdp3.keytab` for gpadmin on hdp3.mydomain.com@DATA.LOCAL:
+   
+   ``` shell
+   PS C:\Users\Administrator> ktpass -princ postgres/hdp3.mydomain.com@DATA.LOCAL -pass abcd1234 -mapuser gpadmin -crypto ALL -ptype KRB5_NT_PRINCIPAL -out hdp3.keytab -kvno 0
+   
+   Targeting domain controller: WIN-TIH2EBEERUK.DATA.LOCAL
+   
+   Using legacy password setting method
+   Key created.
+   Key created.
+   Key created.
+   Key created.
+   Key created.
+   Output keytab to hdp3.keytab:
+   Keytab version: 0x502
+   keysize 61 postgres/hdp3.example.com@DATA.LOCAL ptype 1 (KRB5_NT_PRINCIPAL) vno 0 etype 0x1 (DES-CBC-CRC) keylength 8 (0xa8d0405789a8469e)
+keysize 61 postgres/hdp3.example.com@DATA.LOCAL ptype 1 (KRB5_NT_PRINCIPAL) vno 0 etype 0x3 (DES-CBC-MD5) keylength 8 (0xa8d0405789a8469e)
+keysize 69 postgres/hdp3.example.com@DATA.LOCAL ptype 1 (KRB5_NT_PRINCIPAL) vno 0 etype 0x17 (RC4-HMAC) keylength 16 (0x161cff084477fe596a5db81874498a24)
+keysize 85 postgres/hdp3.example.com@DATA.LOCAL ptype 1 (KRB5_NT_PRINCIPAL) vno 0 etype 0x12 (AES256-SHA1) keylength 32 (0x20648bd82de77bf66a1dcac4b3050bc308f2cc38f4a13e814ad5bd30e67ef388)
+keysize 69 postgres/hdp3.example.com@DATA.LOCAL ptype 1 (KRB5_NT_PRINCIPAL) vno 0 etype 0x11 (AES128-SHA1) keylength 16 (0xa026bb25aa495af5334cd4f185d33071):
+   ```
+   
+10. Transfer the generated keytab file to the HDB master. For example:
+
+    ``` shell
+    root@kdc-server$ scp /etc/ad_keytab/keytabs/hawq.service.keytab <master>:/etc/ad_keytab/keytabs/hawq.service.keytab
+    ```
+Change the ownership of the keytab file to `gpadmin:gpadmin` and the mode to `600`. 
+
+
+    ``` shell
+    root@kdc-server$ ssh <master> chown gpadmin:gpadmin /etc/ad_keytab/keytabs/hawq.service.keytab
+    root@kdc-server$ ssh <master> chmod 600 /etc/ad_keytab/keytabs/hawq.service.keytab
+    ```
+    
+***Can you use automated kerberos setup on Ambari?**
+
+11. Log into HAWQ through the administrator window and install the Kerberos packages on the HAWQ master. 
+
+12. Change the ticket renewal interval.
+
+13. Edit the `.etc/krb5.conf` configuration file to define the Kerberos realm for the cluster. 
+
+   **Can you set up with PS C:\Users\Administrator> ? What if you use Ambari? This page has commands for Powershell: 
+   https://hortonworks.com/blog/enabling-kerberos-hdp-active-directory-integration/**
```

Comment:
We can remove reference of this link as it's kind of out of date and do not have too much valuable for our settings.